schemas/maas-backend/push-notification/request.json

```diff
@@ -46,7 +46,14 @@
               "minItems": 1,
               "maxItems": 1,
               "items": {
-                "$ref": "http://maasglobal.com/core/components/units.json#/definitions/identityId"
+                "anyOf": [
```

Comment:
"oneOf" would be more appropriate in here

Reply:
For reason I'm not quite putting my head around yet, oneOf doesn't work here while anyOf does. And from the documentation I read that the difference is anyOf could match more than 1. With uuid and old cognito identityId being mutually exclusive I do not see it matching both. But, argument also be made about why would oneOf do not work, and I currently do not have an answer ..